charts/influxdb/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: influxdb
-version: 4.10.4
+version: 4.10.5
 appVersion: 1.8.6
```

Comment:
bump this to 1.8.10

Reply:
or skip that and let's merge https://github.com/influxdata/helm-charts/pull/416 first?

Reply:
> or skip that and let's merge #416 first?

#416 first, I'll re-bump this after.

Reply:
done.

Reply:
rebase and then let's merge